### ISSUE TITLE
Manganato delimiter issue on adding

### ISSUE THREAD

**Issue by C9Glax** [metafates/mangal#151]
### Describe the bug

`TRPCClientError: Cannot find the A Returner's Magic Should Be Special.`
Any title containing the characters `'`, `-` and/or `?` can not be added with the above error on Manganato.

### Operating System

Linux

**Comment by eswiig** [comment]
Yes, this desperately needs to be fixed, I think ~ also causes error.
